CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+
+## 0. Status
+
+FreeCAD is in a transition period. The following are to be regarded as GUIDELINES for contribution submission and acceptance. For historical reasons, the actual process MAY diverge from this process during the transition.  Such deviations SHOULD be noted and discussed whenever possible.
+
+## 1. Goals
+
+The FreeCAD Contribution Process is expressed here with the following specific goals in mind:
+
+1. To provide transparency and fairness in /the contribution process.
+2. To allow contributions to be included as quickly as possible.
+3. To preserve and improve the code quality while encouraging appropriate experimentation and risk-taking.
+4. To minimize dependence on individual Contributors by encouraging a large pool of active Contributors.
+5. To be inclusive of many viewpoints and to harness a diverse set of skills.
+6. To provide an encouraging environment where Contributors learn and improve their skills.
+7. To protect the free and open nature of the FreeCAD project.
+
+## 2. Fundamentals
+
+1. FreeCAD uses the git distributed revision control system.
+2. Source code for the main application and related subprojects is hosted on github.com in the FreeCAD organization, hereafter referred to as 'the Platform'.
+3. Problems are discrete, well-defined limitations or bugs.
+4. FreeCAD uses the Platform's issue-tracking system to track problems and contributions.  For help requests and general discussions, use the project forum. 
+5. Contributions are sets of code changes that resolve a single problem.
+6. FreeCAD uses the Pull Request workflow for evaluating and accepting contributions.
+
+## 3. Roles
+1. "User": An member of the wider FreeCAD community who uses the software.
+2. "Contributor":  A person who submits a contribution that resolves a previously identified problem. Contributors do not have commit access to the repository unless they are also Maintainers. Everyone, without distinction or discrimination, SHALL have an equal right to become a Contributor.
+3. "Maintainer": A person who merges contributions. Maintainers may or may not be Contributors. Their role is to enforce the process. Maintainers have commit access to the repository.
+4. "Administrator": Administrators have additional authority to maintain the list of designated Maintainers.
+
+## 4. Licensing, Ownership, and Credit
+1. FreeCAD is distributed under the Lesser General Public License, version 2, or superior (LGPL2+).  Additional details can be found in the LICENSE file.
+2. All contributions to FreeCAD MUST use a compatible license.
+3. All contributions are owned by their authors unless assigned to another.
```

Comment:
You pretty much duplicated my original draft.  The point of the rule is to cover the case where copyright isn't explicitly stated.  In that case, it belongs to the author.  
I removed the second part because it begs the question, "Can I assign my copyright to someone else? Software conservancy?  Blender Org?  Microsoft?" 
The answer is that the copyright owner can assign it to anyone they wish.

Reply:
@yorikvanhavre Can I resolve this?